base/system/diskpart/diskpart_msg.mc

```diff
@@ -6143,40 +6162,40 @@ Example:
     SELECT DISK=PCIROOT(0)#PCI(0100)#ATA(C00T00L01)
 .
 Language=Romanian
-    Selects the specified disk and shifts the focus to it.
+    Selectează discul specificat și mută marcajul pe el.
 
-Syntax:  SELECT DISK=<N>
+Sintaxă: SELECT DISK=<N>
          SELECT DISK=SYSTEM
          SELECT DISK=NEXT
-         SELECT DISK=<Path>
+         SELECT DISK=<Cale>
 
     DISK=<N>
-                The DiskPart disk index number of the disk to receive
-                focus.
+                Numărul indexului discului DiskPart al discului care urmează
+                să primească marcajulul.
 
-    DISK=<Path>
-                The location path of the disk to receive focus.
+    DISK=<Cale>
+                Calea locației discului care urmează să primească marcajulul.
 
     DISK=SYSTEM
-                On BIOS machines, BIOS disk 0 will receive focus.
-                On EFI machines, the disk containing the ESP partition
-                used for the current boot will receive focus. On EFI
-                machines, if there is no ESP, or there is more than
-                one ESP present, or the machine is booted from Windows PE,
-                the command will fail.
+                La calculatoarele cu BIOS, discul 0 din BIOS va fi marcat.
+                La calculatoarele EFI, discul care conține partiția ESP
+                folosită pentru inițializarea curentă va primi marcaj.
+                La calculatoarele EFI, dacă nu este ESP sau este prezent
+                mai mult decât un ESP sau calculatorul este inițializat din
+                Windows PE, comanda va eșua.
 
     DISK=NEXT
-                Once a disk is selected, this command is used to iterate
-                over all disks in the disk list. The next disk in the list
-                will receive focus. If the next disk is the start of the
-                enumeration, the command will fail and no disk will have
-                focus.
+                Odată ce discul e selectat, această comandă este folosită
+                pentru a se repeta peste toate discurile din lista de discuri.
+                Următorul disc din listă va primi marcaj. Dacă următorul
+                disc este începutul enumerării, comanda va eșua și niciun disc
+                nu va trebui să se marcheze.
```

Comment:
Please use the [passive form, not reflexive one](https://github.com/reactos/reactos/blob/master/media/doc/Romanian%20translation%20notes.txt#L49) (i.e. "să fie marcat" instead of "să se marcheze").